dbchanges/reviews.sql

```diff
@@ -0,0 +1,152 @@
+USE foody;
+
+DROP TABLE IF EXISTS foody.restaurants;
+CREATE TABLE foody.restaurants (
+  restaurantId int AUTO_INCREMENT PRIMARY KEY,
+  restaurantName varchar(200),
+  restaurantAddress varchar(200),
+  restaurantPhone varchar(200),
+  restaurantWebsite varchar(200),
+  postDate datetime NOT NULL DEFAULT CURRENT_TIMESTAMP()
+);
```

Comment:
Capitalize MySQL keywords 

DROP TABLE IF EXISTS foody.restaurants;
CREATE TABLE foody.restaurants (
  restaurant_id INT(6) AUTO_INCREMENT PRIMARY KEY,
  restaurant_name VARCHAR(200) NOT NULL,
  restaurant_address VARCHAR(200) NOT NULL,
  restaurant_phone VARCHAR(200),
  restaurant_website VARCHAR(200),
  post_date datetime NOT NULL DEFAULT CURRENT_TIMESTAMP()
);